File: C_Programing/Competitive/Problems_Solving/ASSIGNMENT_12/Q2.c
//WAP which accept  from user and Check whether  it contains 0 in it or not
#include<stdio.h>
#include<stdbool.h>
#define TRUE 1
#define FALSE 0

typedef int BOOL;

BOOL ChkZero(int iNo)
{

  if(iNo<0)
  {
    iNo=-iNo;
  }


  int iDigit=0 ;
 while (iNo!=0)
 {
   iDigit=iNo%10;//Digit remainder check and forward in printf 
  
    if(iDigit==0)
   {
    return true;
   }    
   iNo=iNo/10;//540/54/4/5//Digit Seperate logic 
 }
 return false;
   
}


int main()
{
   
      int iValue=0;
      bool bret=FALSE;
    printf("Enter number  : ");
    scanf("%d",&iValue);


 bret =ChkZero(iValue);
  
 if(bret==true)
 {
  printf("It Contains Zero");
 }
 else
 {
  printf("There is No Zero");
 }

    return 0;
}

File: java_Programing/Competitive/Problems_Solving/ASSIGNMENT_17/Q2.java
//WAP to check whether a number is a pallindrome or not..

import java.util.Scanner;

class Logic
{
    void CheckPalindrome(int No)
    {
        int Temp = No;
        int Rev = 0;
        int Digit = 0;

        while(No != 0)
        {
            Digit = No % 10;//pallindrome logic s
            Rev = (Rev * 10) + Digit;
            No = No / 10;
        }

        if(Temp == Rev)
        {
            System.out.println("Number is Palindrome");
        }
        else
        {
            System.out.println("Number is Not Palindrome");
        }
    }
}

class Q2
{
    public static void main(String A[])
    {
       int iValue=0;

        Scanner sobj=new Scanner(System.in);

        Logic obj = new Logic();
          
        System.out.println("Enter number :");
        iValue=sobj.nextInt();


        obj.CheckPalindrome(iValue);
    }
}